readme.md

```diff
@@ -183,6 +183,7 @@ Made with Electron.
 - [Netron](https://github.com/lutzroeder/netron) - Visualizer for deep learning and machine learning models.
 - [Ao](https://github.com/klauscfhq/ao) - Unofficial Microsoft To-Do app.
 - [Etcher](https://github.com/resin-io/etcher) - Flash OS images to SD cards and USB drives.
+- [Notable](https://github.com/fabiospampinato/notable) - The markdown-based note-taking app that doesn't suck.
```

Comment:
The description should be an objective description, not a marketing tagline. Markdown is a name and should have the first character uppercased.

Reply:
Updated :+1: